vocabularies/Analytics.json

```diff
@@ -133,8 +166,42 @@
                 "Common.Label"
             ]
         },
+        "AnalyticalContext": {
+            "$Kind": "Term",
+            "$Collection": true,
+            "$Type": "Analytics.AnalyticalContextType",
+            "$AppliesTo": [
+                "EntitySet",
+                "Collection"
+            ],
+            "@Core.AppliesViaContainer": true,
+            "@Core.Description": "Collection of properties that define an analytical context"
+        },
+        "AnalyticalContextType": {
+            "$Kind": "ComplexType",
+            "Property": {
+                "$Type": "Edm.PropertyPath",
+                "@Core.Description": "Property that is part of the analytical context"
+            },
+            "Dimension": {
+                "$Type": "Core.Tag",
+                "$DefaultValue": true,
+                "@Core.Description": "The property holds the key of a dimension"
+            },
+            "Measure": {
+                "$Type": "Core.Tag",
+                "$DefaultValue": true,
+                "@Core.Description": "The property holds the numeric value of a measure"
+            },
+            "AccumulativeMeasure": {
+                "$Type": "Core.Tag",
+                "$DefaultValue": true,
+                "@Core.Description": "The measure has non-negative and additive values; it can be used in whole-part charts, e.g. the Donut"
+            }
+        },
         "$Annotations": {
             "Aggregation.CustomAggregate": {
+                "@Core.LongDescription#Analytics": "A property path to an `Aggregation.CustomAggregate` annotation can be used to reference the dynamic property holding the value of the custom aggregate.",
```

Comment:
What's this?

Trying to inject "custom" documentation into a foreign vocabulary term?

Reply:
Yes. Is this not allowed?

Reply:
It has no effect: do you plan to extend the markdown generator to also consult other vocabularies for "remote-injected" descriptions?

If not, better add this to some place in this vocabulary that already shows up in the markdown.